Projects/BricKuber/brickuber_lib.py

```diff
@@ -232,13 +267,19 @@ def Moves(self, cmds):
     
     # Use the camera to read the RGB colors for each of the 9 squares on the face
     def CameraReadFaceColors(self, face):
-        commands.getstatusoutput(('raspistill -w 300 -h 300 -t 1 -o /tmp/BricKuber_%s_face.jpg' % face))
+        debug_print_commands("START: Read Face Colors: " + str(face))
+        # commands.getstatusoutput(('raspistill -w 300 -h 300 -t 1 --sharpness -100 --awb auto --imxfx cartoon -o /tmp/BricKuber_%s_face.jpg' % face))
+        commands.getstatusoutput(('raspistill -w 300 -h 300 -t 1 --imxfx cartoon -o /tmp/BricKuber_%s_face.jpg' % face))
```

Comment:
Does this add a "cartoon" camera filter? If the lighting is right, this change shouldn't be necessary.

Reply:
Seemed to make it work better by flattening out the colors, make it more reliable.  Others might find it hard to get the lighting right (I did!)